styles.css

```diff
@@ -103,6 +103,243 @@ header {
   list-style-type: none;
 }
 
+/* Work Section */
+.work {
+  margin-top: 5rem;
+  padding-bottom: 5rem;
+}
+
+.work .heading {
+  text-align: center;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 400;
+  font-style: normal;
+  font-family: 'Crete Round', serif;
+  margin: 0 auto 2rem auto;
+  color: #1c1a19;
+}
+
+.line {
+  margin: 1rem 1rem 4rem;
+}
+
+.projects-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto;
+  gap: 15px;
+}
+
+.project {
+  margin: 0 auto;
+  max-width: 90%;
+  display: flex;
+  flex-direction: column;
+}
+
+.image-container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.image-container img {
+  width: 100%;
+}
+
+.sub-heading {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-style: normal;
+  line-height: 44px;
+  font-weight: 400;
+  margin: 1rem 1rem 0.5rem;
+  color: #091e42;
+}
+
+.description {
+  font-family: 'Inter', serif;
+  color: #091e42;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  font-style: normal;
+  margin: 0 1rem;
+}
+
+.tags {
+  list-style-type: none;
+  margin: 1rem;
+  display: flex;
+  max-width: 250px;
+  justify-content: space-between;
+}
+
+.lang-tag {
+  border: 1px solid #8993a4;
+  outline: none;
+  font-family: 'Inter', serif;
+  font-size: normal;
+  font-size: 15px;
+  color: #091e42;
+  padding: 0.5rem;
+  margin-right: 0.7rem;
+}
+
+.btn {
+  height: 48px;
+  width: 124px;
+  background-color: #ff6b00;
+  color: white;
+  border: none;
+  margin-left: 1rem;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  font-family: 'Inter', serif;
+}
+
+.btn:hover {
+  background-color: #fff5e1;
+  color: #f55800;
+  border: 1px solid #f55800;
+}
+
+.btn:active {
+  background-color: #e05304;
+  color: white;
+}
+
+.style2 {
+  background-image: url(./assets/project-image-placeholder.png);
+  background-repeat: no-repeat;
+  background-color: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
+  background-size: 100%;
+}
+
+.sub-heading2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: white;
+  margin: 5rem 1.5rem 0.5rem 2rem;
+}
+
+.style2 .description {
+  color: white;
+  margin: 0 1rem 1rem 2rem;
+}
+
+.tags-2 {
+  display: flex;
+  list-style-type: none;
+  margin: 0 2rem 1rem;
+}
+
+.tags-2 .lang-tag {
+  background-color: rgba(255, 255, 255, 0.24);
+  color: white;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  border: none;
+}
+
+.style2 .btn {
+  width: 100%;
+  margin: 1rem 0 0;
+  text-align: center;
+}
+
+/* About Me */
+.aboutme {
+  background-color: #1c1a19;
+  padding-top: 5rem;
+  padding-bottom: 5rem;
```

Comment:
Same here, instead of repeating values you could consider using the [shorthand property `padding: top/bottom right/left`](https://www.w3schools.com/css/css_padding.asp#:~:text=has%20two%20values%3A-,padding%3A%2025px%2050px,-%3B) :+1:

Reply:
Thanks, I have worked on it.